lib/shortcuts/stream-lambda.js

```diff
@@ -25,7 +26,14 @@ const Lambda = require('./lambda');
  *     S3Bucket: 'my-code-bucket',
  *     S3Key: 'path/to/code.zip'
  *   },
- *   EventSourceArn: cf.getAtt('MyStream', 'Arn')
+ *   EventSourceArn: cf.getAtt('MyStream', 'Arn'),
+ *   FilterCriteria: {
+ *     Filters: [
+ *       {
+ *         Pattern: JSON.stringify({ eventName: ['INSERT'] }),
+ *       }
+ *     ]
+ *   }
```

Comment:
I think a separate example with a pattern may be better.

Reply:
Added

Reply:
Please run the doc update command so it can be reflected in the docs